test/mpi/util/mtest_common.c

```diff
@@ -18,7 +18,7 @@
 */
 MPI_Aint MTestDefaultMaxBufferSize()
 {
-    MPI_Aint max_size = 2147483648;
+    MPI_Aint max_size = 1073741824;
```

Comment:
It is odd to have an arbitrarily looking constant in the code. Can we either round it to `1000000000` or use hex form `0x40000000`? It is just a nitpick. I am approving the review regardless of the change.